src/app/core/ui/navigation/navigation/navigation.component.spec.ts

```diff
@@ -103,11 +111,39 @@ describe("NavigationComponent", () => {
 
     mockConfigService.getConfig.and.returnValue(testConfig);
     mockConfigUpdated.next(null);
+    tick();
 
     expect(component.menuItems).toEqual([
       new MenuItem("Children", "child", "/child"),
     ]);
-  });
+  }));
+
+  it("should add menu items where entity permissions are missing", fakeAsync(() => {
```

Comment:
The title of this tests sounds counter-intuitive - why should it add where permissions are missing? and the actual test case looks more like it is simply filtering out those that do not pass the guard